src/components/AnnotationViewer.vue

```diff
@@ -1534,7 +1534,8 @@ export default class AnnotationViewer extends Vue {
         break;
       case "tagging":
         if (
-          this.selectedToolConfiguration.values.action.value === "tag_click"
+          this.selectedToolConfiguration.values.action.value === "tag_click" ||
```

Comment:
Usually `["tag_click", "untag_click"].includes(this.selectedToolConfiguration.values.action.value)` is more readable